patches/tModLoader/Terraria/ModLoader/Mod.cs

```diff
@@ -414,7 +414,7 @@ public void AddConfig(string name, ModConfig mc)
 		/// </summary>
 		/// <param name="key">The key for the ModTranslation. The full key will be Mods.ModName.key</param>
```

Comment:
> The full key will be Mods.ModName.key

From the looks of it, this isn't true anymore (and should probably be changed). This is just from a quick glance though, tell me if I'm wrong.

Reply:
Oh oops, guess I missed that.